src/sentry/audit_log/register.py

```diff
@@ -258,33 +258,36 @@
 )
 default_manager.add(
     AuditLogEvent(
-        event_id=120, name="MONITOR_ADD", api_name="monitor.add", template="Monitor added"
+        event_id=120, name="MONITOR_ADD", api_name="monitor.add", template="added monitor {name}"
     )
 )
 default_manager.add(
     AuditLogEvent(
-        event_id=121, name="MONITOR_EDIT", api_name="monitor.edit", template="Monitor edited"
+        event_id=121, name="MONITOR_EDIT", api_name="monitor.edit", template="edited monitor {name}"
     )
 )
 default_manager.add(
     AuditLogEvent(
-        event_id=122, name="MONITOR_REMOVE", api_name="monitor.remove", template="Monitor removed"
+        event_id=122,
+        name="MONITOR_REMOVE",
+        api_name="monitor.remove",
+        template="removed monitor {name}",
     )
 )
 default_manager.add(
     AuditLogEvent(
         event_id=123,
         name="MONITOR_ENVIRONMENT_REMOVE",
         api_name="monitor.environment.remove",
-        template="Monitor environment removed",
+        template="removed an environment from monitor {name}",
```

Comment:
Should we have the name of the environment?

Reply:
yea, we should but currently we don't store that data with the audit log so could do that in a follow up possibly

Reply:
👍